src/main/java/teammates/ui/webapi/QueryLogsAction.java

```diff
@@ -52,13 +65,19 @@ public JsonResult execute() {
         String nextPageToken = getRequestParamValue(Const.ParamsNames.NEXT_PAGE_TOKEN);
         String traceId = getRequestParamValue(Const.ParamsNames.QUERY_LOGS_TRACE);
         String actionClass = getRequestParamValue(Const.ParamsNames.QUERY_LOGS_ACTION_CLASS);
-        String googleId = getRequestParamValue(Const.ParamsNames.STUDENT_ID);
-        String regkey = getRequestParamValue(Const.ParamsNames.REGKEY);
-        String email = getRequestParamValue(Const.ParamsNames.QUERY_LOGS_EMAIL);
         String logEvent = getRequestParamValue(Const.ParamsNames.QUERY_LOGS_EVENT);
         String sourceLocationFile = getRequestParamValue(Const.ParamsNames.QUERY_LOGS_SOURCE_LOCATION_FILE);
         String sourceLocationFunction = getRequestParamValue(Const.ParamsNames.QUERY_LOGS_SOURCE_LOCATION_FUNCTION);
         String exceptionClass = getRequestParamValue(Const.ParamsNames.QUERY_LOGS_EXCEPTION_CLASS);
+        String googleId = null;
+        String regkey = null;
+        String email = null;
+
+        if (userInfo.isAdmin && !userInfo.isMaintainer) {
```

Comment:
No need for `!userInfo.isMaintainer`. Admins can always include this information regardless of whether they are maintainer or not.

Reply:
Ok. It is updated.